src/TCommandLine.h

```diff
@@ -121,6 +124,8 @@ public slots:
     char** mpSystemSuggestionsList = nullptr;
     char** mpUserSuggestionsList = nullptr;
     QSet<QString> commandLineSuggestions;
+    QSet<QString> tabCompleteBlacklist;
```

Comment:
### Note A

I don't know if the coder of the `commandLineSuggestions` considered this, but it also applies to `tabCompleteBlacklist` in that this is local to THIS `TCommandLine` instance so if the user has any additional command lines set up for their game then these things are going to be independent for each command line - which is probably NOT what is wanted. TBH If I was using this I might even have a separate "command lines" put aside to enter the names of friends and foes in.

As a separate coding note these are also `public` so having dedicated getters and setters introduces pointless calling overheads - the callers could just as easily call the Qt methods in those methods directly.